cmd/controller/app/database/mongodb/compute.go

```diff
@@ -111,3 +111,56 @@ func (db *MongoService) UpdateComputeStatus(computeId string, computeStatus open
 
 	return updateTime, nil
 }
+
+func (db *MongoService) CheckComputeByRegion(region string) ([]string, error) {
```

Comment:
The function name is misleading. How about the following?
```suggestion
func (db *MongoService) GetComputesByRegion(region string) ([]string, error) {
```